tests/MoBi.Tests/Core/SBML/AssignmentImporterSpecs.cs

```diff
@@ -63,4 +63,23 @@ public void Compartment_InitialAssignmentCreationTest()
          }
       }
    }
+
+   public class InitialRulesAssignmentImporterTests : ContextForSBMLIntegration<RuleImporter>
+   {
+      protected override void Context()
+      {
+         base.Context();
+         _fileName = Helper.TestFileFullPath("tiny_example_12.xml");
+      }
+
+
+      [Observation]
+      public void FormulaParameter_InitialAssignmentCreationTest()
+      {
+         var psvbb = _moBiProject.ParametersStartValueBlockCollection.FirstOrDefault();
+         psvbb.ShouldNotBeNull();
+         psvbb.Where(psv => psv.Formula != null).Each(psv => psv.Formula.ObjectPaths.Each((p => p.Resolve<IFormulaUsable>(psv).ShouldNotBeNull())));
```

Comment:
even here.... it makes sense to simplify some of  the query to be more readable.